app/views/claims/claim_school.html.erb

```diff
@@ -1,9 +1,59 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
+
+    <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
+
     <h1 class="govuk-heading-xl">
       Which school were you employed at between 6 April XXXX and 5 April XXXX?
     </h1>
 
-    <p class="govuk-body">Coming next...</p>
+    <% if @schools.blank? %>
+
+      <% if params[:school_search].present? %>
+        <p class="govuk-body">
+          <strong>No results match that search term. Try again.</strong>
```

Comment:
If I perform a search for 'pen' I see the validation errors but also 'No results match that search term' - this could be confusing to users. Could we check that they did a valid search before showing this message?

Reply:
Great spot! My thinking here is because the validation for this search is in the controller and if the validation fails we add an error to the current claim I can do:

`<% if params[:school_search].present? && current_claim.errors.empty? %>`

Rather than duplicating the length check in the view, what do you think?

Reply:
@mec that sounds like a sensible approach to me.